src/vm/method.cpp

```diff
@@ -2149,12 +2146,29 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
 
     // We create stable entrypoints for these upfront
     if (IsWrapperStub() || IsEnCAddedMethod())
+    {
+        if (IsZapped() && !HasStableEntryPoint())
+        {
+            // If the method is zapped but doesn't have a stable entry point, then it must have
+            // a zapped precode. That precode is not patchable, so we can't make it a stable entry point.
+            // So instead force creation of runtime allocated precode which is patchable and which will
+            // become the stable entry point.
+            // TODO: How to solve the problem of patching all possible MT slots for this method.
```

Comment:
`MethodDesc::DoBackpatch` will keep looking for more unpatched slots every time it gets called. We just need to make sure that it works as expected for these.